tests/cal_display_test.py

```diff
@@ -137,6 +137,23 @@ def test_str_week():
     '    20 21 22 23 24 25 26 ',
     '\x1b[1mM\xe4r \x1b[0m27 28 29  1  2  3  4 ']
 
+example_de_bsd = [
```

Comment:
I do think that it's clearer the way it is currently written (two separate variables).  It makes the dot version explicitly related to bsd, so the bread crumb trail is easier to follow.

If anything, I'd define a set of OS-specific abbrevations, e.g.:

```python
default_de_months = [ "Jan ", "Feb ", "M\xe4r ", ...]
bsd_de_months = [ "Jan.", "Feb.", "M\xe4rz", ...]
```

then substitute them into `example_de` and `example_de_bsd`.